src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/invalidjavadocposition/InputInvalidJavadocPosition.java

```diff
@@ -0,0 +1,47 @@
+package /** violation */ com.puppycrawl.tools.checkstyle.checks.javadoc.invalidjavadocposition;
+
+/** violation */
+import java.lang.String;
+
+/** violation */
+/** valid */
+class InputInvalidJavadocPosition {
+    /** violation */    
+}
+/* ignore */
+class InputInvalidJavadocPosition2 {
```

Comment:
please add testscase:
```
/** valid */
/* ignore */
class InputInvalidJavadocPosition2 {

/** violation */
/** valid */
class InputInvalidJavadocPosition2 {
```

Reply:
Second item is already there above `InputInvalidJavadocPosition`.
First item was added.